ModuleConfig.cfc

```diff
@@ -43,6 +43,53 @@ component {
             .initArg( name = "returnFormat", value = "array" );
     }
 
+    function onUnload(){
+        structDelete( application, "quickMeta" );
+    }
+
+    /**
+     * This interception ensures that quick entity parent/child relationships are available to discriminated entities
+     * @event          The current request context
+     * @interceptData  The intercept data for `afterAspectsLoad`.
+     */
+    function afterAspectsLoad( event, interceptData ){
```

Comment:
The problem with the using the metadata cache is that it expires - because it uses the `default` cache.  I thought about creating a custom quick cachebox cache that was scoped in to the application.  Ultimately, though, I don't want anything in this metadata to ever be evicted, because I need the parent entities to be able to access it at all times.   Storing it in the app scope guarantees it's always there.